src/screens/FeedScreen/elements/Search.js

```diff
@@ -103,7 +103,7 @@ const styles = StyleSheet.create({
     paddingTop: 7,
     paddingBottom: 7,
     borderBottomWidth: 1,
-    borderBottomColor: COLORS.lightgrey
+    borderBottomColor: COLORS.gray100
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in border color from `COLORS.lightgrey` to `COLORS.gray100` should be fine as long as it doesn't compromise the visibility of the border. Ensure that the new color maintains sufficient contrast with the background.
```diff
-    borderBottomColor: COLORS.lightgrey
+    borderBottomColor: COLORS.gray100
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->